cmd/gokeyless/gokeyless.go

```diff
@@ -95,8 +95,13 @@ func main() {
 		os.Exit(0)
 	}
 
+	// If we make it here we need to ask for user input, so we need to give up
+	// and log an error instead (in case the server is running as a daemon).
+	// Failing hard with an error message makes the problem obvious, whereas a
+	// daemon blocked waiting on input can be hard to debug.
 	if needNewCertAndKey() {
-		initializeServerCertAndKey()
```

Comment:
so seems like it will not do auto-bootstrapping at all?

Reply:
The intention is to only auto-bootstrap when the ‘-config-only’ flag is provided

Reply:
sure, got it!